Composer/packages/form-dialogs/src/atoms/types.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type PropertyRequiredKind = 'required' | 'optional';
+
+export type FormDialogPropertyKind = 'ref' | 'number' | 'integer' | 'string' | 'array';
+
+export type RefPropertyPayload = TypedPropertyPayload & {
+  ref: string;
+};
+
+export type TypedPropertyPayload = {
+  kind: FormDialogPropertyKind;
+  entities?: string[];
+};
+
+const builtInFormats = ['data-time', 'date', 'time', 'email', 'uri', 'iri'] as const;
```

Comment:
what's `data-time` ?

Reply:
Type of date-time (based on builtInStringFormats below)

Reply:
typo indeed